src/framework/uicomponents/qml/Muse/UiComponents/SeparatorLine.qml

```diff
@@ -28,6 +28,7 @@ Rectangle {
     id: root
 
     property int orientation: prv.parentIsHorizontal ? Qt.Vertical : Qt.Horizontal
+    property int separatorWidth: 1
```

Comment:
Late to the party, but `thickness` might have been a nice unambiguous name

Reply:
@cbjeukendrup `thickness` sounds much better. I'll push another PR before it gets used elsewhere.